config/config.go

```diff
@@ -97,6 +98,18 @@ func parsePlugins(pluginList []interface{}) ([]*PluginConfig, error) {
 	return plugins, nil
 }
 
+func (c *Config) getInterface(ver protocolVersion) (string, error) {
+	directive := fmt.Sprintf("server%d.interface", ver)
```

Comment:
Nit: should probably validate IPv4/6 so a random integer can't be passed here.

Reply:
d'oh, I forgot to replicate that check from `getListenAddress`. I could probably define a new `ipVersion` type and define two constants

Reply:
added a `protoVersionCheck` function